scripts/install_pr.sh

```diff
@@ -1 +1,2 @@
-echo "Not Ready"
\ No newline at end of file
+echo "Not Ready"
```

Comment:
What are we doing with this script that does nothing?

Reply:
Keep in mind, scripts/install_bx.sh checks if the build is a pull request event already.